RevenueCatUI/Helpers/Package+VariableDataProvider.swift

```diff
@@ -142,6 +166,46 @@ extension Package: VariableDataProvider {
 
 // MARK: - Private
 
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, *)
+private extension Package {
+
+    func priceEndsIn00Cents(_ priceString: String) -> Bool {
```

Comment:
Its a little more swifty to start a function that returns a boolean with something like `is` 🤷‍♂️ 

```suggestion
    func isPriceEndingIn00Cents(_ priceString: String) -> Bool {
```

Reply:
I made the change, but the use of "is" here feels off to me, I think because it's operating on the passed in object rather than the object that it's being called on.